website/docs/configuration/functions/timestamp.html.md

```diff
@@ -13,7 +13,7 @@ description: |-
 earlier, see
 [0.11 Configuration Language: Interpolation Syntax](../../configuration-0-11/interpolation.html).
 
-`timestamp` returns the current date and time.
+`timestamp` returns the current date and time in UTC.
```

Comment:
Maybe we can adopt some of the older verbiage from the [0.11 documentation](https://www.terraform.io/docs/configuration-0-11/interpolation.html) and then remove the paragraph below this line.

```suggestion
`timestamp` returns a UTC timestamp string in [RFC 3339](https://tools.ietf.org/html/rfc3339) format.
```